assets/js/components/RtcPlayer.tsx

```diff
@@ -26,10 +27,32 @@ const RtcPlayer = ({ isMyself, name, presenterStreams, streamsAvailable }: RtcPl
   return (
     <div>
       <video width={1000} height={700} autoPlay ref={videoRef} />
-      <audio ref={audioRef} />
+      {!isMyself && <audio ref={audioRef} />}
       <h5>{isMyself ? name + " (Me)" : name}</h5>
     </div>
   );
 };
 
+type RtcClientPlayerProps = {
+  name: string;
+  clientStreamAvailable: SourceType;
+};
+
+export const RtcClientPlayer = ({ name, clientStreamAvailable }: RtcClientPlayerProps) => {
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  useEffect(() => {
+    if (name in presenterStreams && clientStreamAvailable && videoRef.current != null) {
```

Comment:
It's a pretty complex condition, move it to a helper function.

Reply:
I honestly thing it would add even more complexity